Event page for hub cloud costs Community Call (#181)
Openscapes/website-new: events/2025-04-22-community-call-hub-cloud-costs/index.qmd

---

> title: "Community Call: Beyond “it depends”: what we’re learning about cloud costs for Earth science workflows in our JupyterHub"

Reviewer: ```suggestion
title: "Community Call: What we’re learning about cloud costs for Earth science workflows in our JupyterHub"
```
As the title has evolved/improved, I think the "beyond it depends" might make it too long? I first suggested that thinking it was punchy, and am also not tied to it. What do you think Stef?

> description: something?

Reviewer: ```suggestion
description: Beyond "it depends" - learning about cloud costs for science by building reusable open tooling for monitoring and reporting
```

Reviewer: This is one idea; a little redundant or long maybe?

Reviewer 2: I like it a lot!

------------------------------------------------------------------------

::::: grid
::: g-col-4
![](cloud-costs.png){fig-alt="A diagram illustrating a user accessing the cloud from their laptop. The image includes icons representing a cloud computing instance and cloud storage. There is a price attached to each of the cloud icons." fig-align="left" width="100%"}
:::

::: g-col-8
Openscapes and our partners at NASA, NOAA, and 2i2c have been learning together about monitoring and managing the costs of cloud computing in a JupyterHub. We would like to share some of these lessons with our community.
:::
:::::

**Date**: Tuesday, April 22, 2025  
**Time**: 9:30 - 10:30 am PT ([find your local time](https://www.timeanddate.com/worldclock/fixedtime.html?msg=Community+Call%3A+Openscapes+Goes+to+the+White+House&iso=20240926T13&p1=1050&ah=1))  
**Where**: Zoom   
**Register (free)** via [Zoom](https://zoom.us/meeting/register/_SSlgeuuToiLNH5p5VZSXw) to get the meeting link <!--- **View** [recording](https://youtu.be/uKAm3uEyUks) and [event summary](/blog/2024-10-03-openscapes-recognized-white-house/). --->

> Many organizations are using cloud computing for computational analysis and teaching workshops, often using JupyterHubs as the platform for this work. Tracking usage and attributing costs to specific users and workflows can be tricky on these shared hubs, and we have been learning some strategies and tools to help us understand them. The purpose of this Community Call is to share what we've learned: tools and processes to explore cloud costs, as well as figures like the basic costs for hosting a hub, cost per user, cost per science workflow, and what it costs to run a workshop in the hub. Openscapes Director Julie Lowndes will interview our guests: Andy Teucher (Openscapes), Alex Lewandowski (NASA Alaska Satellite Facility), Yuvi Panda (2i2c). Participants will leave with new insights and we'll post a recording after the event.

Reviewer: ```suggestion
Many organizations are using cloud computing for computational analysis and teaching workshops, often using JupyterHubs as the platform for this work. Tracking usage and attributing costs to specific users and workflows can be tricky on these shared hubs, and we have been learning some strategies and tools to help us understand them. The purpose of this Community Call is to share what we've learned: tools and processes to explore cloud costs, as well as figures like the basic costs for hosting a hub, cost per user, cost per science workflow, and what it costs to run a workshop in the hub. Through a light interview structure, Andy Teucher (Openscapes), Alex Lewandowski (NASA Alaska Satellite Facility), and Yuvi Panda (2i2c) will share their experiences and what they are building. Participants will leave with new insights and we'll post a recording after the event.
```

What would **you** like to ask them?

### Speakers

[Andy Teucher](https://openscapes.org/team) is a core Openscapes team member and develops software and cloud infrastructure.

Alex Lewandowski is a Software Engineer at the [NASA Alaska Satellite Facility (ASF)](https://www.earthdata.nasa.gov/centers/asf-daac) data center, University of Alaska Fairbanks Geophysical Institute.

[Yuvi Panda](https://2i2c.org/author/yuvi-panda/) is the technical lead and co-founder of [2i2c](https://2i2c.org/).

> [Julie Lowndes](https://openscapes.org/team) is Openscapes founding director and core team member.

Reviewer: ```suggestion
```
I'm ok without centering me; better to focus on others here I think

### Resources

-   [openscapes.cloud](https://openscapes.cloud/) for NASA and NOAA Fisheries JupyterHub policies and [cost reporting](https://openscapes.cloud/usage-reporting.html).
-   [2i2c](https://2i2c.org/) helps communities build their own interactive computing hub in the cloud with open infrastructure.

> -   [jupycost](https://github.com/openscapes/jupycost): A work-in-progress R

Reviewer: @ateucher would it make sense to link to Sean Kross/Hutch's aws package too? Or any other community resources?

Reviewer 2: Yes for sure! I wasn't sure how deep to go but yes I will
